exporter/opentelemetry-exporter-jaeger-thrift/examples/jaeger_exporter_example.py

```diff
@@ -0,0 +1,46 @@
+import time
+
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger import thrift
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+trace.set_tracer_provider(TracerProvider())
+tracer = trace.get_tracer(__name__)
+
+# create a JaegerExporter
+jaeger_exporter = thrift.JaegerExporter(
+    service_name="my-helloworld-service",
+    # configure agent
+    agent_host_name="localhost",
+    agent_port=6831,
+    # optional: configure also collector
+    # collector_endpoint="/api/traces?format=jaeger.thrift",
```

Comment:
Can we mention the full default collector endpoint for thrift i.e, `http://<collector-hostname>:14268/api/traces?format=jaeger.thrift` ?

Reply:
Done